near-sdk/src/collections/vec/mod.rs

```diff
@@ -0,0 +1,258 @@
+//! A vector implemented on a trie. Unlike standard vector does not support insertion and removal
+//! of an element results in the last element being placed in the empty position.
+// TODO update these docs
+
+mod impls;
+mod iter;
+
+use self::iter::{Iter, IterMut};
+use crate::collections::append_slice;
+use crate::{env, CacheCell, CacheEntry, EntryState, IntoStorageKey};
+use borsh::{BorshDeserialize, BorshSerialize};
+use std::collections::{btree_map::Entry, BTreeMap};
+use std::ptr::NonNull;
```

Comment:
I am personally not a huge fan of mashing all the imports together for a new code. I'd written this as 

```rust
use std::collections::{btree_map::Entry, BTreeMap};
use std::ptr::NonNull;

use borsh::{BorshDeserialize, BorshSerialize};

use crate::collections::append_slice;
use crate::{env, CacheCell, CacheEntry, EntryState, IntoStorageKey};
use crate::collections::vec::iter::{Iter, IterMut};
```

Might make sense to add a code style section to the contributing.md, and say "for the new code, we use the following import order / don't care about import order"

Reply:
Yeah, I was thinking about that, I just figured I'd keep it as is (which I didn't see as bad because it's already grouped as its lexicographical) until someone shared an opinion on the style.